person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name: 'Unknown', parent_permission: 'Unknown')
+    @id = Ramdom.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    @parent_permission == 'allowed'
+  end
```

Comment:
- [ ] One of the requirements says that the `can_use_services?` method should be public the only private method for the Person class is `of_age?`. Kindly remove the `can_use_services?` method in the private.

Reply:
This issue is well resolved! ✔️